src/gui/tray/Window.qml

```diff
@@ -304,6 +351,9 @@ Window {
                             Layout.preferredHeight: Style.accountAvatarSize
                             Layout.preferredWidth: Style.accountAvatarSize
 
+                            Accessible.role: Accessible.Graphic
+                            Accessible.name: qsTr("Current user avatar")

```

Comment:
Avatars are decorative so in the web interface they usually have _empty_ alt text instead of "User avatar".

@tyrylu @Adriorjalesvidal what would be expected here? Just no Accessible.name set, or a specific Accessible.role besides "Accessible.graphic"?